{"cve_repaire_en": "", "cve_code_exp_url": "", "cve_id": "CVE-2019-13611", "cve_categories": ["category1", "category2"], "cve_code_detect_url": "", "cve_name": "cve_name", "cve_risk_level": "determine", "cve_detail_en": "An issue was discovered in python-engineio through 3.8.2. There is a Cross-Site WebSocket Hijacking (CSWSH) vulnerability that allows attackers to make WebSocket connections to a server by using a victim's credentials, because the Origin header is not restricted.", "cve_name_en": "cve_name_en", "edit_time": "Sat Oct 12 16:04:12 2019", "cve_code_des_url": "", "cve_other_info": "", "modify_time": "2019-07-15", "cve_detail": "", "references": [{"URL": "https://github.com/miguelgrinberg/python-engineio/issues/128", "Description": "MISC:https://github.com/miguelgrinberg/python-engineio/issues/128"}], "cve_repaire": "", "editor": "warcup", "add_time": "2019-07-15"}